config/development.json

```diff
@@ -11,7 +11,8 @@
     "sourceMaps": true,
     "prettyPrint": false,
     "watchExpressions": false,
-    "search": true
+    "search": true,
+    "conditionalBreakpoints": true
```

Comment:
I don't think we need to put this behind a flag. Let's just land features and make sure they work from the start. If there are major bugs I think we should fix in the the first PR. But I doubt there are major bugs, there might just be UX improvements we can do. That's the point of landing in nightly though; we have time until uplift to tweak it to get it great.